node/pkg/gwrelayer/gwrelayer.go

```diff
@@ -171,15 +267,32 @@ func shouldPublish(payload []byte, targetChain vaa.ChainID, targetAddress vaa.Ad
 	return true, nil
 }
 
+// shouldPublishToTokenBridge returns true if a message should be forwarded to the token bridge, false if not.
+func shouldPublishToTokenBridge(tokenBridges tokenBridgeMap, v *vaa.VAA) bool {
+	if _, exists := tokenBridges[tokenBridgeKey{emitterChainId: v.EmitterChain, emitterAddr: v.EmitterAddress}]; !exists {
+		return false
+	}
+
+	if len(v.Payload) == 0 {
+		return false
+	}
+
+	if v.Payload[0] != 2 {
```

Comment:
Is there a const we can use here for VAA payload type instead of using `2` as a magic number?

Reply:
I am not aware of them being defined anywhere. I added a comment though.